trillian/testdata/int-ca.cfg

```diff
@@ -23,6 +23,12 @@ subjectKeyIdentifier = hash
 authorityKeyIdentifier = keyid:always,issuer
 keyUsage = critical, digitalSignature, nonRepudiation, keyEncipherment, dataEncipherment, keyAgreement, encipherOnly, decipherOnly
 
+[ v3_user_serverAuth ]
+subjectKeyIdentifier = hash
+authorityKeyIdentifier = keyid:always,issuer
+keyUsage = critical, digitalSignature, nonRepudiation, keyEncipherment, dataEncipherment, keyAgreement, encipherOnly, decipherOnly
```

Comment:
Nit/curiosity: was it intentional to have all these extra KU on here, or would `digitalSignature` be enough?

Reply:
Not intentional, I just copy-pasted the `v3_user` option set from above, and added the EKU.